pages/common/gh-ssh-key.md

```diff
@@ -0,0 +1,20 @@
+# gh ssh-key
+
+> Manage GitHub SSH keys from the command line.
+> More information: <https://cli.github.com/manual/gh_ssh-key>.
+
+- Display help:
+
+`gh ssh-key`
+
+- List SSH keys:
```

Comment:
```suggestion
- List SSH keys attached to your account:
```

Reply:
Why not something like:
> List SSH keys for the currently authenticated user:

Reply:
Sounds good.  I'll update my suggestions.